core/server/data/migrations/versions/4.3/08-migrate-members-signup-setting.js

```diff
@@ -0,0 +1,96 @@
+const logging = require('../../../../../shared/logging');
+const ObjectId = require('bson-objectid');
+
+module.exports = {
```

Comment:
Not a big deal at all, but you could have used `createTransactionalMigration` from the utils for ever so slightly more concise migration :relaxed:

Reply:
➕ on this 🙂

Reply:
Updated to use `createTransactionalMigration` 🙂